platforms/qurt/src/px4/SerialImpl.cpp

```diff
@@ -251,8 +251,9 @@ ssize_t SerialImpl::write(const void *buffer, size_t buffer_size)
 	int ret_write = qurt_uart_write(_serial_fd, (const char *) buffer, buffer_size);
 
 	if (ret_write < 0) {
-		PX4_ERR("%s write error %d", _port, ret_write);
-
+		if (errno != EAGAIN) {
```

Comment:
Qurt does not set errno so shouldn't be checked here

Reply:
can you push the fix?